### ISSUE TITLE
Feature request: ability to add left labels

### ISSUE THREAD

**Issue by MDLC01** [pauladam94/curryst#2]
Currently, labels can only be added to the right of rules, with the `name` argument.

I would like to also be able to add labels to the left of rules. This could be done by adding a `left-label` or `llabel` argument.

**Comment by pauladam94** [comment]
Good idea. Please make suggestion on this repo [Curryst](https://github.com/pauladam94/curryst).

**Comment by pauladam94** [comment]
Good idea but it has to be :
- a non breaking change with the previous API (because it is a minimal API)
- we can think about a "left-name" parameter that is optional. If it is used, then it is the name of the label use and it changes the type set.

**Comment by MDLC01** [comment]
> * we can think about a "left-name" parameter that is optional. If it is used, then it is the name of the label use and it changes the type set.

Does this mean you would not be able to have both a left name and a right name for a single rule?

**Comment by MDLC01** [comment]
Maybe it could be an additional boolean argument, then? Something like `name-on-left`?
